app/components/ucsf/RefinementLists/ucsfEligibilitiesMap.ts

```diff
@@ -22,13 +22,6 @@ interface UcsfEligibilityMap {
   [key: string]: EligibilityGroup[];
 }
 
-// N.B.: Until we have found a way to make use of Algolia's "Show More" refinements button,
```

Comment:
Turns out this comment was moot as we are using this the eligibilities in this file to filter out those returned by Algolia anyhow